sky/utils/common_utils.py

```diff
@@ -352,3 +353,8 @@ def remove_file_if_exists(path: str):
     except FileNotFoundError:
         logger.debug(f'Tried to remove {path} but failed to find it. Skip.')
         pass
+
+
+def is_wsl() -> bool:
+    """Detect if running under WSL"""
+    return 'microsoft' in platform.uname()[3].lower()
```

Comment:
https://www.scivision.dev/python-detect-wsl/?

Reply:
This doesn't work on my machine:
```
>>> uname().release
'4.4.0-19041-Microsoft'
```